Dockerfile

```diff
@@ -15,15 +15,14 @@ ARG ARCH="amd64"
 ARG K3S_ROOT_VERSION="v0.8.1"
 ADD https://github.com/rancher/k3s-root/releases/download/${K3S_ROOT_VERSION}/k3s-root-xtables-${ARCH}.tar /opt/xtables/k3s-root-xtables.tar
 RUN tar xvf /opt/xtables/k3s-root-xtables.tar -C /opt/xtables
-ARG TAG="v0.13.0-rancher1"
-ARG PKG="github.com/coreos/flannel"
-ARG SRC="github.com/rancher/flannel"
-RUN git clone --depth=1 https://${SRC}.git $GOPATH/src/${PKG}
-WORKDIR $GOPATH/src/${PKG}
+ARG TAG="v0.14.0"
+ARG SRC="github.com/flannel-io/flannel"
```

Comment:
Please retain **both** the SRC and PKG args as this allows for this repo to be a ready-made build+packaging harness for forks.

Reply:
ok. it is fine if they have the same value, right?